crates/re_renderer/shader/point_cloud.wgsl

```diff
@@ -13,6 +13,8 @@ var color_texture: texture_2d<f32>;
 struct BatchUniformBuffer {
     world_from_obj: Mat4,
     flags: u32,
+    _padding: UVec2, // UVec3 would take its own 4xf32 row, UVec2 is on the same as flags
```

Comment:
why do we need to pad at all?

Reply:
It's either here or in Rust. If we put nothing here they get on the same "row" since alignment for UVec2 is 8. https://www.w3.org/TR/WGSL/#alignment-and-size

This then translates to
    ```
    pub struct BatchUniformBuffer {
        pub world_from_obj: wgpu_buffer_types::Mat4,
        pub flags: u32,                   // PointCloudBatchFlags
        pub outline_mask_ids: glam::Vec2,

        pub end_padding: [wgpu_buffer_types::PaddingRow; 16 - 5],
    }
    ```
But `bytemuck` doesn't like this because that implies 32bit padding after `outline_mask_ids` and it forces all padding to be explicit. So we end up with

    ```
    pub struct BatchUniformBuffer {
        pub world_from_obj: wgpu_buffer_types::Mat4,
        pub flags: u32,                   // PointCloudBatchFlags
        pub outline_mask_ids: glam::Vec2,
        pub _padding: u32,

        pub end_padding: [wgpu_buffer_types::PaddingRow; 16 - 5],
    }
    ```